viewing_party/main.py

```diff
@@ -0,0 +1,151 @@
+#Wave 1 
+def create_movie(title,genre,rating): 
+    if (title and genre and rating): 
+        movie = {}
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
+    elif not title or not genre or not rating:
+        return None
+  
+def add_to_watched(user_data,movie): 
+    user_data["watched"].append(movie)
+    return user_data 
+
+def add_to_watchlist(user_data,movie):
+    if movie not in user_data["watchlist"]: 
+        user_data["watchlist"].append(movie)
+    return user_data 
+
+def watch_movie(user_data, title):
+
+    for e in user_data["watchlist"]:
+        if e["title"] == title:
+            user_data["watchlist"].remove(e) 
+            user_data['watched'].append(e)
+    return user_data 
+ 
+
+# Wave # 2
+
+def get_watched_avg_rating(user_data): 
+    watched = user_data["watched"]
+    if len (watched) == 0:  
+        return 0 
+
+    total = 0 
+    for  movie in watched:  
+        total += movie["rating"]
+    
+    average = total/len(watched)
+    return average 
+
+def get_most_watched_genre(user_data): 
+    most_watched = user_data["watched"]
+
+    all_genres = {}
+    
+    for movie in most_watched: 
+        if movie["genre"] not in all_genres: 
+            all_genres [movie["genre"]] = 1
+
+        else: 
+            all_genres [movie["genre"]] += 1
```

Comment:
Style: Remove blank lines in front of the `else` or `elif` keywords:

```suggestion
        if movie["genre"] not in all_genres: 
            all_genres [movie["genre"]] = 1
        else: 
            all_genres [movie["genre"]] += 1
```

It makes it not appear visually "connected" to the `if`.